app/controllers/sessions_controller.rb

```diff
@@ -2,6 +2,15 @@ class SessionsController < ApplicationController
 
   def show
     @cart_items = session[:cart].map {|id, num| [Item.find(id), num]}
+    @total_price = total_price
+  end
+
+  private
+
+  def total_price
+    @cart_items.sum do |item, num|
```

Comment:
Unused block argument - `num`. If it's necessary, use `_` or `_num` as an argument name to indicate that it won't be used.